### PR TITLE
Cleanup gemspec

### PR DESCRIPTION
**Pull request by ashmaroli**
## Summary
- Gem date is automatically computed when the gem is built. (Ref: [Source Code](https://github.com/rubygems/rubygems/blob/17778fe2bba64d7cf4157b69c356aa382449d27f/lib/rubygems/specification.rb#L1750-L1757))
- Reduce line-length with a heredoc. (Ref: [Official Guide](https://guides.rubygems.org/specification-reference#description))
- Rearrange some of the elements for cleaner alignment

### CODE REVIEW

File: jekyll-paginate-v2.gemspec [hunk 1]
```diff
@@ -2,26 +2,30 @@
...
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'jekyll-paginate-v2/version'
-require 'date'
 
 Gem::Specification.new do |spec|
   spec.name          = "jekyll-paginate-v2"
   spec.version       = Jekyll::PaginateV2::VERSION
   spec.platform      = Gem::Platform::RUBY
-  spec.required_ruby_version = '>= 2.0.0'  # Same as Jekyll
-  spec.date          = DateTime.now.strftime('%Y-%m-%d')
```
Comment (sverrirs) [inline jekyll-paginate-v2.gemspec]: I need to add this specifically to get the date correct, please be absolutely certain that this is not needed anymore.

Comment (ashmaroli) [reply jekyll-paginate-v2.gemspec]: I've pushed a couple of gems myself.
But for your satisfaction, you may *try the following steps*:
1. Checkout this branch locally
2. Bump the plugin version to any greater number (e.g.: `2.0.1`) (you do not have to commit)
3. Cut a new gem:
    ```
    gem build jekyll-paginate-v2.gemspec
    ```
4. Install the newly cut gem (w/o pushing to remote):
    ```
    gem install jekyll-paginate-v2 --local
    ```
5. Manually inspect the installed specification file in your system's gem directory.
    e.g. `/sverrirs/Ruby24/lib/ruby/gems/2.4.0/specifications/jekyll-paginate-v2-2.0.1.gemspec`

Comment (ashmaroli) [reply jekyll-paginate-v2.gemspec]: Additionally, I've linked to the "Official Documentation" in the opening post. They a list of "Required Attributes" and "Optional Attributes"

Comment (ashmaroli) [reply jekyll-paginate-v2.gemspec]: Additional reference: [Rubygems source code](https://github.com/rubygems/rubygems/blob/d43e4d010551fcd856222e8a0fa6bb1878b00c4f/lib/rubygems/specification.rb#L1731-L1741)

**Pull request closed without merging**